spec/factories/doctors_notes.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :doctors_note do
+    passenger_id 1
+    expiration_date "2017-07-27"
+    overriden_by 1
```

Comment:
Curious what this column represents. If it's a user, then I'd recommend defining an association that way in your model file.